backend/lobby/consumers.py

```diff
@@ -101,8 +104,36 @@ def gamelog_send_handler(self, data):
 
         self.send_server_response_to_client(username, room, mud_gamelog_input)
 
+    def selected_card_handler(self, data):
+        username = data['user']
+        room = data['room']
+
+        print("selected card handler")
+
+        payload = data['payload']
+
+        self.send_server_response_to_client(username, room, payload)
+
+    def get_the_three_store_cards(self):
+        card_market = self.master_game_controller.deck_handler.store
+        # card_market is now a list of three random cards, modify as needed
+        card1 = card_market[0]
+        card2 = card_market[1]
+        card3 = card_market[2]
+        return card_market
+
+    # This test is temporary to prove the GameConsumer has access to the card store
+    def test_get_store_cards():
+        game = GameConsumer(WebsocketConsumer("test"))
+        three_cards = game.get_the_three_store_cards()
+        print(
+            "\n\tCARD 1:{}\n\tCARD 2:{}\n\tCARD 3:{}".format(three_cards[0].name, three_cards[1].name, three_cards[2].name))
+        assert len(three_cards) == 3
+    
     commands = {
         'init_user_request': init_chat_handler,
         'gamelog_send_request': gamelog_send_handler,
-        'selected_dice_request': selected_dice_handler
+        'selected_dice_request': selected_dice_handler,
+        'send_cards_request': get_the_three_store_cards
```

Comment:
please rename 'get_the_three_store_cards' to get_the_three_store_cards_handler

Reply:
Renamed the handler